composer/loggers/tqdm_logger.py

```diff
@@ -117,7 +117,8 @@ def init(self, state: State, logger: Logger) -> None:
 
     def _start(self, state: State):
         assert self.is_train is not None, "self.is_train should be set by the callback"
-        total_steps = len(state.train_dataloader) if self.is_train else len(state.eval_dataloader)
+        total_steps = len(state.train_dataloader) if self.is_train else sum(
```

Comment:
For TQDM, does the total_steps include evaluation, or do we want to have evaluation be part of a separate TQDM instance? Curious for @ravi-mosaicml 's thought here too

Reply:
Total steps should be the number of times TQDM is updated. Currently this is done on each batch. IDK, could go either way on this -- don't have a strong preference on the TQDM format.